manifests/repo/webtatic.pp

```diff
@@ -4,7 +4,7 @@
 #
 class yum::repo::webtatic {
   $osver = split($::operatingsystemrelease, '[.]')
-  $mirrorlist = $osver[0] ? {
+  $mirrorlist = (0+$osver[0]) ? {
     5 => 'http://repo.webtatic.com/yum/centos/5/$basearch/mirrorlist',
     6 => 'http://repo.webtatic.com/yum/el6/$basearch/mirrorlist',
     7 => 'http://repo.webtatic.com/yum/el7/$basearch/mirrorlist',
```

Comment:
Cleaner solution would be to use `$::operatingsystemmajrelease` IMHO. Try to use existing facts when possible.

And quote the version numbers. 👍